src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+/* #region stars */
```

Comment:
remove comments all over the page

Reply:
ok. but this is not a comments it's regions)